docs/querying/filters.md

```diff
@@ -35,263 +35,221 @@ Apache Druid supports the following types of filters.
+  ]
+}
+```
 
-Note that an "insensitive_contains" search is equivalent to a "contains" search with "caseSensitive": false (or not
-provided).
+### OR
 
-#### Fragment
+| Property | Description | Required |
+| -------- | ----------- | -------- |
+| `type` | Must be "or".| Yes |
+| `fields` | List of filter JSON objects, such as any other filter defined on this page or provided by extensions. | Yes |
 
-|property|description|required?|
-|--------|-----------|---------|
-|type|This String should always be "fragment".|yes|
-|values|A JSON array of String values to run the search over.|yes|
-|caseSensitive|Whether strings should be compared as case sensitive or not. Default: false(insensitive)|no|
+#### Example: equivalent of `WHERE someColumn = 'a' OR otherColumn = 1234 OR anotherColumn IS NULL`
 
-## In filter
+``` json
+{
+  "type": "or",
+  "fields": [
+    { "type": "equality", "column": "someColumn", "matchValue": "a", "matchValueType": "STRING" },
+    { "type": "equality", "column": "otherColumn", "matchValue": 1234, "matchValueType": "LONG" },
+    { "type": "null", "column": "anotherColumn" } 
+  ]
+}
+```
 
-In filter can be used to express the following SQL query:
+### NOT
 
-```sql
- SELECT COUNT(*) AS 'Count' FROM `table` WHERE `outlaw` IN ('Good', 'Bad', 'Ugly')
-```
+| Property | Description | Required |
+| -------- | ----------- | -------- |
+| `type` | Must be "not".| Yes |
+| `field` | Filter JSON objects, such as any other filter defined on this page or provided by extensions. | Yes |
 
-The grammar for a "in" filter is as follows:
+#### Example: equivalent of `WHERE someColumn IS NOT NULL`
 
 ```json
-{
-    "type": "in",
-    "dimension": "outlaw",
-    "values": ["Good", "Bad", "Ugly"]
-}
+{ "type": "not", "field": { "type": "null", "column": "someColumn" }}
 ```
 
-The "in" filter supports the use of extraction functions, see [Filtering with Extraction Functions](#filtering-with-extraction-functions) for details.
-
-If an empty `values` array is passed to the "in" filter, it will simply return an empty result.
-
-If the `dimension` is a multi-valued dimension, the "in" filter will return true if one of the dimension values is
-in the `values` array.
 
-If the `values` array contains `null`, the "in" filter matches null values. This differs from the SQL IN filter, which
-does not match NULL values.
+## In filter
+The in filter can match input rows against a set of values, where a match occurs if the value is contained in the set.
 
-## Like filter
+| Property | Description | Required |
+| -------- | ----------- | -------- |
+| `type` | Must be "in".| Yes |
+| `dimension` | Input column or virtual column name to filter. | Yes |
+| `values` | List of string value to match. | Yes |
+| `extractionFn` | [Extraction function](./dimensionspecs.md#extraction-functions) to apply to `dimension` prior to value matching. See [Filtering with Extraction Functions](#filtering-with-extraction-functions) for details. | No |
```

Comment:
```suggestion
| `extractionFn` | [Extraction function](./dimensionspecs.md#extraction-functions) to apply to `dimension` prior to value matching. See [Filtering with extraction functions](#filtering-with-extraction-functions) for details. | No |
```

Reply:
You could just do a search and replace all for the sentence case fix; I didn't comment on all of them